### PR TITLE
center and button color

### PR DESCRIPTION
**Pull request by JuSchlup**
Finally!!!  💯

### CODE REVIEW

File: app/views/buddyships/new.html.erb [hunk 1]
```diff
@@ -23,7 +23,7 @@
                   <hr>
                   <p class="text-muted m-t-15"><%= buddy.goal_description %></p>
 
-                  <div class="btn btn-success"><%= link_to "Select", buddyships_path(buddy: buddy.id), method: "POST" %></div>
+                  <div class="btn btn"><%= link_to "Select", buddyships_path(buddy: buddy.id), method: "POST" %></div>
```
Comment (ymik0410) [inline app/views/buddyships/new.html.erb:26]: Juli, does just class="btn" (not "btn btn") work here as well?

**Review by ymik0410 — CHANGES_REQUESTED** [review]
Juli, could you please check the comment for the button?

File: app/views/buddyships/new.html.erb (hunk 1 above)
Comment (JuSchlup) [reply app/views/buddyships/new.html.erb:26]: If we were working only with CSS, btn would work. btn btn is a Bootstrap code.

Comment (ymik0410) [reply app/views/buddyships/new.html.erb:26]: ok

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -9,66 +9,67 @@
...
-    <div class="card-body">
-      <p class="card-text">Select one of the top 3 best suited accountability
-    buddies to get paired with.</p>
-    </div>
-  </div>
-  <div class="card">
-    <img src="/assets/goals-target2.png" alt="">
-    <div class="card-body">
-      <p class="card-text">Connect with your buddy and show up everyday to motivate
-      and encourage each Other.</p>
```
Comment (ymik0410) [inline app/views/pages/home.html.erb:32]: each other

**Review by ymik0410 — CHANGES_REQUESTED** [review]

**Review by ymik0410 — COMMENTED** [review]
Thanks!

**Review by ymik0410 — APPROVED** [review]

**Pull request merged**